src/main/java/io/jenkins/pluginhealth/scoring/scores/DeprecatedPluginScoring.java

```diff
@@ -29,39 +29,25 @@
 import io.jenkins.pluginhealth.scoring.model.ResultStatus;
 import io.jenkins.pluginhealth.scoring.model.ScoreResult;
 import io.jenkins.pluginhealth.scoring.probes.DeprecatedPluginProbe;
+import io.jenkins.pluginhealth.scoring.probes.UpdateCenterPluginPublicationProbe;
 
-import io.jenkins.pluginhealth.scoring.model.updatecenter.UpdateCenter;
-import io.jenkins.pluginhealth.scoring.service.ProbeService;
-import io.jenkins.pluginhealth.scoring.service.UpdateCenterService;
 
 import org.springframework.stereotype.Component;
 
-import java.io.IOException;
-
 @Component
 public class DeprecatedPluginScoring extends Scoring {
     private static final float COEFFICIENT = .8f;
     private static final String KEY = "deprecation";
 
-    final ProbeContext probeContext;
-    private final ProbeService probeService;
+
 
     @Override
     protected ScoreResult doApply(Plugin plugin) {
         final ProbeResult deprecatedPluginProbeResult = plugin.getDetails().get(DeprecatedPluginProbe.KEY);
 
+        final ProbeResult doesPluginExistInUpdateCenterMapResult = plugin.getDetails().get(UpdateCenterPluginPublicationProbe.KEY);
```

Comment:
Hey @alecharp 

I am do not completely understand this ask. I have asked a similar question in Plugin_Health_Score Gitter channel.

Reply:
Can you please describe the question you have regarding this line?

Reply:
Hey @dheerajodha 

I did not understand this part: `.getDetails().get(...)`.

I could not find where `getDetails()` is defined or which Java library it belongs too.  IntelliJ did not help me understand that.

Reply:
In order to understand where `getDetails()` is defined, you need to observe that it is getting called by `plugin` object, which has access to this method that you're trying to find.